module1.js

```diff
@@ -53,7 +62,9 @@ function question3(number) {
 */
 
 function question4(array) {
-  // TODO:
+  for(var i = 0; array.length > i; i+2){
+      return array[i];
```

Comment:
This function is meant to return an array.
Here, you're returning just one element in the array, and also, multiple times, because it's within a loop.
Figure out how to get every 2nd element in the input array - and you don't have to do this with loops! :)